sympy/integrals/transforms.py

```diff
@@ -1826,10 +1827,23 @@ def _collapse_extra(self, extra):
         return plane, cond
 
     def _try_directly(self, **hints):
+        noconds = hints.get('noconds', False)
         fn = self.function
-        debug('----> _try_directly: %s'%(fn, ))
         t_ = self.function_variable
         s_ = self.transform_variable
+        r = expand(fn, deep=False)
+        if r.is_Add and noconds:
+            # Try apply rules to all sum terms; if it does not work, use
+            # a combined way
+            st = [S.Zero]
+            for f in r.args:
+                LT = _laplace_apply_rules(f, t_, s_, **hints)
+                if LT is None:
+                    st = None
+                    break
+                st.append(LT)
+            if not st is None:
+                return fn, Add(*st)
```

Comment:
I have to say I don't quite understand how the `_try_directly` method fits with everything else e.g. why it's a different method from `_compute_transform` and why `LaplaceTransform._try_directly` is different from `IntegralTransform._try_directly`.

Reply:
We introduced this when we did the rule-based Laplace transform, but I don't rememeber why we thought it was necessary or good to do it.  Should `_try_directly` be the standard and everything else should go to `_compute_transform`?

Reply:
> Should `_try_directly` be the standard and everything else should go to `_compute_transform`?

That would probably make more sense.

Reply:
I just came up with a third option: give the `IntegralTransforms` a new method that can be called with `self._use_rules`, and rewrite `doit` such that it does this before `_try_directly`.

Of course I would do this in a way that changes nothing at all unless a derived integral transform overloads `_use_rules`.

This might also make it easier to make other integrals rule-based later.

What do you think?

Reply:
I guess that's good. I haven't looked through this code in as much detail as you probably.

Reply:
probably :)))))

Starting work now ...